docs/topics/plugin-icon-legend.adoc

```diff
@@ -5,17 +5,17 @@
 [id='plugin-icon-legend_{context}']
 = About {ProductShortName} issue icons
 
-In the Issue Explorer and file editors, {ProductShortName} issues use an icon to indicate their severity level and status. The following table describes the meaning of the various icons.
+{ProductShortName} displays icons in *Issue Explorer* and file editors to indicate the severity and status of issues related to migration. The table below describes the meaning of these icons.
 
 .{ProductShortName} issue icons
 [cols="20%a,80%",options="header",]
 |====
 |Icon |Description
 |image::error.png[Mandatory] |The issue is mandatory to fix for a successful migration.
 |image::info.gif[Optional] |The issue is optional to fix for migration.
```

Comment:
That's an info icon. I'm not sure that the description in the text is accurate. Might be good to check with Phil. Is it really an issue or is it information? It's hard to tell without examples. Hey, that would also be a good idea, if one could add examples, but that is probably out of scope for this. So you can pursue this or not.